micro-deps/micro-deps/src/main/groovy/com/ofg/infrastructure/discovery/watcher/DependencyState.java

```diff
@@ -0,0 +1,6 @@
+package com.ofg.infrastructure.discovery.watcher;
+
+enum DependencyState {
```

Comment:
Yes. Why not? I know it was public, but as far as I remember It was only used in this package (despite tests, but spock handles that)